<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="settingsLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>100</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="layoutLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Layout</string>
   </property>
  </widget>
  <widget class="QLabel" name="speedLayout">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Solving Speed</string>
   </property>
  </widget>
  <widget class="QComboBox" name="layoutComboBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Cube Projection</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Two Cubes</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="speedComboBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Step through</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1/2 second</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1 second</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2 second</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>164</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::SaveAll</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
